### ISSUE TITLE
Terraform docs site is out of date

### ISSUE THREAD

**Issue by xsmaster** [hashicorp/terraform#4085]
https://www.terraform.io/docs/providers/template/index.html

Shows the old syntax. 

```
# Template for initial configuration bash script
resource "template_file" "init" {
    filename = "init.tpl"

    vars {
        consul_address = "${aws_instance.consul.private_ip}"
    }
}
```

Looks like someone beat me to the punch to make the fix. 
https://github.com/hashicorp/terraform/commit/f0292d420fcb653ca18d60aa1513f577f29d1593 

When will the site get updated?

**Comment by stack72** [comment]
@xsmaster the docs site usually gets pushed around the same time as a release. I reckon within a few weeks

**Comment by henning** [comment]
odd enough, here it's right: https://www.terraform.io/docs/providers/template/r/file.html

Maybe this is not yet changed in master?

**Comment by xsmaster** [comment]
I thought I saw that the master was updated. I could be wrong.

**Comment by henning** [comment]
Sorry, checked it only after asking - master is right! https://github.com/hashicorp/terraform/blob/master/website/source/docs/providers/template/index.html.markdown

**Comment by stack72** [comment]
@xsmaster  so I think that this can be closed then?

**Comment by xsmaster** [comment]
No worries

**Comment by henning** [comment]
But 0.6.7 release branch is stil wrong https://github.com/hashicorp/terraform/blob/release/v0.6.7/website/source/docs/providers/template/index.html.markdown

**Comment by xsmaster** [comment]
Sure! Thanks for your help

**Comment by phinze** [comment]
This should be fixed with the v0.6.8 website deploy from today. :+1:

**Comment by xsmaster** [comment]
Woot!!!! Thanks!
